pootle/apps/pootle_store/models.py

```diff
@@ -1794,9 +1830,14 @@ def insert(i1, j1, j2, offset):
                     new_unit_index = i1 + index + 1 + offset
                     uid = unit.getid()
                     if uid not in old_unitid_set:
-                        self.addunit(unit, new_unit_index, user=system,
-                                     revision=update_revision)
-                        changes['added'] += 1
+                        # Don't add unit if store revision is less than max
+                        # unit revision
+                        add_unit = (store_revision is None
+                                    or max_unit_revision <= store_revision)
+                        if add_unit:
```

Comment:
The comment already states what the condition does. I think it is ok to directly use the condition instead of creating a temporary variable.